plugins/woocommerce/client/admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -237,100 +304,126 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
-										<span className="woocommerce-marketplace__product-card__vendor" />
-									</p>
+					<div className="woocommerce-marketplace__product-card__meta">
+						<h2 className="woocommerce-marketplace__product-card__title">
+							<CardLink />
+						</h2>
+						{ showVendorLoading && (
+							<p className="woocommerce-marketplace__product-card__vendor-details">
+								<span className="woocommerce-marketplace__product-card__vendor" />
+							</p>
+						) }
+						{ showVendor && (
+							<p className="woocommerce-marketplace__product-card__vendor-details">
+								{ productVendor && (
+									<span className="woocommerce-marketplace__product-card__vendor">
+										<span>
+											{ __( 'By ', 'woocommerce' ) }
+										</span>
+										{ productVendor }
+									</span>
+								) }
+								{ productVendor && isSponsored() && (
+									<span
+										aria-hidden="true"
+										className="woocommerce-marketplace__product-card__vendor-details__separator"
+									>
+										·
+									</span>
 								) }
-								{ showVendor && (
-									<p className="woocommerce-marketplace__product-card__vendor-details">
-										{ productVendor && (
-											<span className="woocommerce-marketplace__product-card__vendor">
-												<span>
-													{ __(
-														'By ',
-														'woocommerce'
-													) }
-												</span>
-												{ productVendor }
-											</span>
-										) }
-										{ productVendor && isSponsored() && (
-											<span
-												aria-hidden="true"
-												className="woocommerce-marketplace__product-card__vendor-details__separator"
-											>
-												·
-											</span>
-										) }
-										{ isSponsored() && (
-											<span className="woocommerce-marketplace__product-card__sponsored-label">
-												{ __(
-													'Sponsored',
-													'woocommerce'
-												) }
-											</span>
-										) }
-									</p>
+								{ isSponsored() && (
+									<span className="woocommerce-marketplace__product-card__sponsored-label">
+										{ __( 'Sponsored', 'woocommerce' ) }
+									</span>
 								) }
-								{ isCompact && footer }
-							</div>
-						</div>
+							</p>
+						) }
+						{ isCompact && footer }
 					</div>
-					{ showDescription && (
-						<p className="woocommerce-marketplace__product-card__description">
-							{ ! isLoading && decodedDescription }
-						</p>
-					) }
-					{ ! isCompact && footer }
 				</div>
+			</div>
+			{ showDescription && (
+				<p className="woocommerce-marketplace__product-card__description">
+					{ ! isLoading && decodedDescription }
+				</p>
+			) }
+			{ ! isCompact && footer }
+		</div>
+	);
+
+	const CardWrapper = () => {
+		if ( ! isLoading ) {
+			return (
+				<div
+					className="woocommerce-marketplace__product-card-wrapper"
+					style={ {
+						color: 'inherit',
+						textDecoration: 'none',
+					} }
```

Comment:
I don't really understand why we are adding it. I tried disabling it locally but see no changes. Maybe we can remove it?

Reply:
This change was originally made to address a nested links issue, and I want to remember the product title was getting styled as a link as a result of the change.

You're right, that's no longer happening, so a later change likely made this code obsolete. Removing in ae04f82f1e

Reply:
Thanks!